src/components/logic/relationships.py

```diff
@@ -173,12 +173,6 @@ def activity_pair_matches(self, trace, activity1, activity2) -> bool:
 class Next_One_Way (Relationship):
     """Implementation of the next-one-way relationship algorithm."""
 
-    def __init__(self, log):
-        """Store the traces."""
-        super().__init__(log)
-
-        self.mode = Relationship.Mode.EXISTS
-
```

Comment:
Why is that a bug? 

It is actually necessary to override the constructor at this point, so that you don't have to pass in another mode later.